nifi-nar-bundles/nifi-cipher-bundle/nifi-cipher-processors/src/main/resources/docs/org.apache.nifi.processors.cipher.DecryptContentCompatibilityMode/additionalDetails.html

```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+  <!--
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+        http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+  -->
+  <head>
+    <meta charset="UTF-8">
+    <title>DecryptContentCompatibilityMode</title>
+    <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css" />
+  </head>
+  <body>
+    <h2>Summary</h2>
+    <p>
+      This Processor supports decryption using legacy formats and Password-Based Encryption Algorithms.
+      <a href="https://www.rfc-editor.org/rfc/rfc8018">RFC 8018</a> defines the Password-Based Cryptography
```

Comment:
Weird interaction here between iframe and external link.

Reply:
Good point, adding the `target` attribute to the tag should help by opening it in a new window.